### PR TITLE
feat: Remove smartlook and add intercom messenger

### PR DESCRIPTION
**Pull request by webrdaniel**
- Remove Smartlook as we no longer use it
- Adding Intercom messenger script to allow AI chat
 - added a new variable to the Intercom for this

The growth team has to config properly the messenger window for docs in Intercom and give us a green light to deploy this.

### CODE REVIEW

File: apify-docs-theme/src/theme.js [hunk 1]
```diff
@@ -132,6 +132,21 @@ This either means that your Docusaurus setup is misconfigured, or that your GitH
             o.plugins.push(postcssPreset); // allow newest CSS syntax
             return o;
         },
+        injectHtmlTags() {
```
Comment (jbartadev) [inline apify-docs-theme/src/theme.js:135]: Questing: @webrdaniel can't we use the https://www.npmjs.com/package/@gracefullight/docusaurus-plugin-intercom plugin for this?

Comment (webrdaniel) [reply apify-docs-theme/src/theme.js:135]: I was kinda discouraged by only 12 weekly downloads but yeah we can use this 👍

Comment (webrdaniel) [reply apify-docs-theme/src/theme.js:135]: I have some issues getting it working with the plugin. It is producing an invalid script. So I guess we will stick with the manual implementation
Screenshot 2024-09-02 at 11 53 41

**Review by B4nan — APPROVED** [review]

**Review by jbartadev — APPROVED** [review]
I'm cool with that if it's going to work 👍

**Pull request merged**